ui/pages/confirmation/confirmation.js

```diff
@@ -271,6 +272,16 @@ export default function ConfirmationPage({
     ///: END:ONLY_INCLUDE_IF
   ];
 
+  ///: BEGIN:ONLY_INCLUDE_IF(keyring-snaps)
+  if (
+    [...Object.values(SNAP_MANAGE_ACCOUNTS_CONFIRMATION_TYPES)].includes(
```

Comment:
This should still work as `Object.values` returns array
```suggestion
    Object.values(SNAP_MANAGE_ACCOUNTS_CONFIRMATION_TYPES).includes(
```

Reply:
Indeed, good catch!